src/components/attendee/ProfileOverlay/ProfileOverlay.tsx

```diff
@@ -167,13 +175,11 @@ export const ProfileOverlay: React.FC<ProfileOverlayProps> = ({ profile }) => {
           partyNameError={errors?.partyName}
         />
 
-        <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
Why did u change `form` to `div` here?

Reply:
otherwise I was getting `there couldn't be a form as a descendant of form` error due to the change in the `InputSelect`